security.md

```diff
@@ -149,6 +140,16 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
+### Hash your customer's passwords with a proper password hashing function
+
+
+If you can afford it, use a [third party authentication service](#customer-users-management) to handle password storage, password management and password recovery. 

```

Comment:
@itaifrenkel *okta* looks interesting! Is it a hosted platform entirely or middleware as well?